sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/TableDestination.java

```diff
@@ -56,13 +56,22 @@ public TableDestination(String tableSpec, @Nullable String tableDescription,
         timePartitioning != null ? BigQueryHelpers.toJsonString(timePartitioning) : null);
   }
 
+  public TableDestination(TableReference tableReference, @Nullable String tableDescription,
```

Comment:
Note: I hesitated to let this private (but I just wanted to be consistent with the rest of constructors.